src/__tests__/components/FieldComment.spec.tsx

```diff
@@ -24,47 +24,43 @@
 
  */
 
-import React from 'react';
-import renderer from 'react-test-renderer';
-import { theme } from "@looker/components"
-import { ThemeProvider } from "styled-components"
+import React from 'react'
+import { screen } from '@testing-library/react'
+import { renderWithExtensionContext } from '../test_utils/render_with_extension'
 import { FieldComment } from '../../components/FieldComment'
 
-jest.mock("../../components/FieldCommentDisplay", () => ({
-  FieldCommentDisplay: () => "FieldCommentDisplay"
-}))
-
-it('renders correctly', () => {
-  const tree = renderer
-    .create(
-      <ThemeProvider theme={theme}>
-        <FieldComment
-          comment={{
-            pk: "timestamp::author",
-            author: 1,
-            content: "This is my comment.",
-            timestamp: 7171717171,
-            edited: false
-          }}
-          editingComment={""}
-          setEditingComment={()=>{}}
-          setCommentContent={()=>{}}
-          editComment={()=>{}}
-          deleteComment={()=>{}}
-          commentContent={""}
-          field={{}}
-          authorData={{
-            display_name: "Mr. Foo Bar",
-            first_name: "Foo",
-            last_name: "Bar",
-            avatar_url: "imgsrv.com/foo/bar"
-          }}
-          me={{}}
-          addingNew={false}
-          permissions={{}}
-          />
-      </ThemeProvider>
+describe('<FieldMetadata>', () => {
+  it('renders correctly', () => {
+    renderWithExtensionContext(
+      <FieldComment
+        comment={{
+          pk: 'timestamp::author',
+          author: 1,
+          content: 'This is my comment.',
+          timestamp: 7171717171,
```

Comment:
fun with numbers!

Reply:
Blame says it was Noah :D. Maybe something special for him :D. I did notice the describe description so gonna fix that